### PR TITLE
radix.node does not check siblings when first child match part of path.

### PR DESCRIPTION
**Pull request by zxfishhack**
when using route declare below:
```go
r := New()
v1 := r.Group("/v1")
v1.GET("/foo/{id}/{pageSize}/{page}", handle1)
v1.GET("/foo/{id}/{iid}", handle2)
v1.GET("/foo/{id}", handle3)
```
the route `/foo/{id}/{iid}` will never touch.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,7 @@ module github.com/fasthttp/router
 go 1.11
 
 require (
+	github.com/elithrar/simple-scrypt v1.3.0
```
Comment (savsgio) [inline go.mod:6]: Could you remove this line, please?

File: go.sum [hunk 2]
```diff
@@ -1,5 +1,7 @@
 github.com/andybalholm/brotli v1.0.0 h1:7UCwP93aiSfvWpapti8g88vVVGp2qqtGyePsSuDafo4=
 github.com/andybalholm/brotli v1.0.0/go.mod h1:loMXtMfwqflxFJPmdbJO0a3KNoPuLBgiu3qAvBg8x/Y=
+github.com/elithrar/simple-scrypt v1.3.0 h1:KIlOlxdoQf9JWKl5lMAJ28SY2URB0XTRDn2TckyzAZg=
```
Comment (savsgio) [inline go.sum:3]: Could you remove this lines, please?

Comment (zxfishhack) [reply go.sum]: reverted

File: go.mod (hunk 1 above)
Comment (zxfishhack) [reply go.mod]: reverted

**Comment by zxfishhack** [comment]
go.mod/go.sum has been reverted.

**Comment by savsgio** [comment]
Thanks! 😉

**Pull request merged**